runtime/rxjava2-extensions/build.gradle

```diff
@@ -10,4 +10,8 @@ dependencies {
   testImplementation deps.truth
 }
 
+kotlin {
+  archivesBaseName = "sqldelight-rxjava2-extensions"
```

Comment:
These all shouldn't need to be in the `kotlin` block. It's a property on the `Project` (which is the implicit scope at the root).

e.g., https://github.com/JakeWharton/SdkSearch/blob/cf72a43ac00f9ad8722969ce81f691093306e10c/store/js/build.gradle#L3